compiler/rustc_codegen_llvm/src/attributes.rs

```diff
@@ -194,6 +194,16 @@ pub fn apply_target_cpu_attr(cx: &CodegenCx<'ll, '_>, llfn: &'ll Value) {
     );
 }
 
+pub fn apply_tune_cpu_attr(cx: &CodegenCx<'ll, '_>, llfn: &'ll Value) {
```

Comment:
This applies the attribute regardless of whether the option is specified or not. Is there a good reason why we're applying it even if the `-Ztune-cpu` is not specified?

Reply:
LLVM should handle the fallback value, so no real reason for that except testability. I guess I will make a separate thing for it.